eng/common/scripts/update-docs-metadata.ps1

```diff
@@ -43,7 +43,7 @@ function GetMetaData($lang){
     }
   }
 
-  $metadataResponse = Invoke-WebRequest-WithHandling -url $metadataUri -method "GET" | ConvertFrom-Csv
+  $metadataResponse = Invoke-RestMethod -Uri $metadataUri -method "GET" -MaximumRetryCount 3 -RetryIntervalSec 10 | ConvertFrom-Csv
```

Comment:
Was Invoke-WebRequest-WithHandling previously the reason for the `. (Join-Path $PSScriptRoot artifact-metadata-parsing.ps1)` at the top of this file? I didn't see anything else in eng/common/scripts/artifact-metadata-parsing.ps1 being called in here. If this was the only call then we can probably remove it.

Reply:
I did remove, but was blind and didn't see `VerifyPackages`